src/main/java/com/redhat/devtools/lsp4ij/features/documentSymbol/DocumentSymbolData.java

```diff
@@ -37,7 +38,7 @@ public class DocumentSymbolData extends FakePsiElement {
     private final @NotNull PsiFile psiFile;
     private final DocumentSymbolData parent;
     private final @NotNull LanguageServerItem languageServer;
-    private TextRange textRange;
+    private volatile TextRange textRange = null;
```

Comment:
This could be reverted if desired, but I noticed that `textRange` was being lazy-initialized in a manner that's not guaranteed atomic, so I converted it to use DCL.